packages/base-driver/lib/basedriver/helpers.js

```diff
@@ -409,6 +409,20 @@ export function generateDriverLogPrefix(obj, sessionId = null) {
   return `${obj.constructor.name}@${node.getObjectId(obj).substring(0, 4)}`;
 }
 
+/**
+ * Register the session in the context to be used in the logs
```

Comment:
maybe we need to rename this method to reflect its purpose better, since the registered log context is connected to the actual async context, e.g. a request.

Reply:
I have changed it, but do you have any other good method name suggestions?

Reply:
yes, I have mentioned that in the other comment